pages/common/git-commit.md

```diff
@@ -22,3 +22,7 @@
 - Commit only specific (already staged) files:
 
 `git commit {{path/to/file1}} {{path/to/file2}}`
+
+- Commit staged files to the repository with a message while allowing the commit be empty:
+
+`git commit -m "{{message}}" --allow-empty`
```

Comment:
Could you expand the arguments?
```suggestion
`git commit --message="{{message}}" --allow-empty`
```

Reply:
I understand however its an incredible annoyance. Also most users would probably remember 

`m for message`

Reply:
I wouldn't be too sure about that. Newcomers to git prabably don't have to commit that often that typing `--message` would be a problem.

Reply:
At least I'd remove the `=`:
```suggestion
`git commit --message "{{message}}" --allow-empty`
```

Reply:
Hmm, I wonder if with such a common command as `git commit`, we should maybe keep the short options 🤔

I've never used `--message` before if I recall correctly 😅

Reply:
It's the same situation as with #6083. Nobody would ever use `--sync --clean --clean`, but it's still good to know what they do, especially for beginners.

Reply:
yes but them commands are long and multiple flags, this is `m` if I was thinking for a message I would try -m.

The `S` for sync doesn't mean much to me, I'd immediately think search

Reply:
I would suggest that this is a rare exception to the rule. Generally speaking you're going to be using this command a *lot*, so I'd suggest keeping the short version in this case, and maybe updating the description like we have on the `tar` page for example.

Reply:
> Hmm, I wonder if with such a common command as `git commit`, we should maybe keep the short options thinking
> 
> I've never used `--message` before if I recall correctly sweat_smile

You haven't used message or just always used -m?